webrender/src/renderer.rs

```diff
@@ -2215,7 +2217,7 @@ pub struct RendererOptions {
     pub enable_batcher: bool,
     pub render_target_debug: bool,
     pub max_texture_size: Option<u32>,
-    pub workers: Option<Arc<Mutex<ThreadPool>>>,
+    pub workers: Option<Arc<rayon::ThreadPool>>,
```

Comment:
while I use this style in some of the personal project, I believe the convention in WR is to `use` all the types you need